phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/LocalRepository.java

```diff
@@ -39,7 +39,7 @@
 
 public class LocalRepository implements Repository {
     private final static Logger LOGGER = LoggerFactory.getLogger(LocalRepository.class);
-    private static final String CATEGORY_ICON_NAME = "icon.png";
+    private static final String ICON_NAME = "icon.png";
```

Comment:
With the reduced name this constant needs a description. E.g. what does the icon represent, currently I would assume it represents the repository?